### DISCUSSION TITLE
should we create close many tabs from same hosts?

### DISCUSSION THREAD

**Discussion by Adamkaram** [gdh1995/vimium-c#754] [Q&A] [answered]
i've a new idea that i can create a featuer that let us to close tabs that blongs to (x.com)  and x depend on inputs for example 
i have 20 tabs some of them let's say 10 belongs to youtube and  i need to close every tabs with youtube host 
so will be input or we can reuse inputs from (Vomnibar  , find mode ) if it possiable insted of create one for one purpose
when users puts x.com therefore everything belongs to that host will colsed immediately

**Comment by gdh1995** [comment, marked as answer]
Please see the usage said in https://github.com/gdh1995/vimium-c/issues/732#issuecomment-1269960640 and https://github.com/gdh1995/vimium-c/issues/732#issuecomment-1280301176 .

**Comment by Adamkaram** [comment]
yup i see it now thanks
